osu.Game/Overlays/Settings/Sections/GameplaySection.cs

```diff
@@ -35,8 +37,18 @@ private void load(RulesetStore rulesets)
             foreach (Ruleset ruleset in rulesets.AvailableRulesets.Select(info => info.CreateInstance()))
             {
                 SettingsSubsection section = ruleset.CreateSettings();
```

Comment:
Any reason for not covering this also? The `CreateInstance()` call above this is checked by `RulesetStore`, so this looks to be the only one that's not covered here.

Reply:
fair